mattermost.go

```diff
@@ -134,6 +134,7 @@ func main() {
 
 		setDiagnosticId()
 		go runSecurityAndDiagnosticsJob()
+		go resetStatuses()
```

Comment:
Maybe this should run synchronous before the api.StartServer? What if a user connects before this is run?